R/convertIRF.R

```diff
@@ -0,0 +1,29 @@
+#' convertIRF
+#'
+#' Add missing periods to data frame and interpolate missing values.
+#' The ISO codes of "IRF" data are compared with the official ISO code country list.
+#' NA values are replaced with zeros and historical data are corrected.
+#'
+#' @param x MAgPIE object.
+#'
+#' @return The "IRF" data with spatial entries for each country.
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("IRF", convert = TRUE)
+#' }
+#'
+#' @importFrom quitte as.quitte
+#' @importFrom quitte interpolate_missing_periods
+
+convertIRF <- function(x) {
+
+  x <- as.quitte(x) %>%
```

Comment:
shouldn't `%>%` be included in the function documentation?

Reply:
ok